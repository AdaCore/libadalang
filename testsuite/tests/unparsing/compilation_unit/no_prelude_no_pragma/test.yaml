driver: unparser
rule: compilation_unit
description: |
  Test that there are no literal lines before or after the body.
