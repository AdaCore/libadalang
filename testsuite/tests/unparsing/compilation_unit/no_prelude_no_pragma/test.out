package Baz is
   type Qux;
end Baz;
